packages/design-system/src/components/search-suggestion-list/search-suggestion-list.tsx

```diff
@@ -0,0 +1,40 @@
+import { CmpProfileNon } from '../../icons/src';
+
+import * as styles from './search-suggestion-list.css';
+
+interface RelatedKeyword {
+  artistId: string;
+  profileUrl: string;
+  name: string;
+}
+
+interface SearchSuggestionListProps {
+  relatedKeyword: RelatedKeyword[] | undefined;
+}
+
+const SearchSuggestionList = ({
+  relatedKeyword,
+}: SearchSuggestionListProps) => {
+  return (
+    <>
```

Comment:
```suggestion
    <ul>
```
https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element

Reply:
감사합니다~

Reply:
지연시간 limit 모두 임의로 설정해두었습니다~ lodash를 사용하는 것과 react-simplikit 사용하는 것 중에 뭐가 더 좋을지 비교한번 해보겠습니다

Reply:
@seueooo 님의 의견을 보고 찾아보니 lodash가 react-simplikit에 비해서 번들사이즈가 7-8배 정도 크네요...
직접 구현해보거나, react-simplikit를 도입해보는 것도 좋은 방법일듯합니다!

Reply:
직접 구현하는게 낫겠네요 반영하고 다시 푸시하겠습니다

Reply:
`packages/design-system/src/components/search-suggestion-list/search-suggestion-list.tsx` 에서 최상위 태그를 ul 로 변경 한다면 기존에 부모컴포넌트에서 사용하던 `searchSuggestionListSection` 에 스타일 속성을 

```css
export const searchSuggestionListSection = style({
  ...themeVars.display.flexColumn,
  alignItems: 'flex-start',
  gap: '2rem',
  width: '100%',
  height: 'auto',
  padding: '2rem',
});
```
search-suggestion-list.tsx 내에서 지정해주어야 하는데, 저번에 제가 슬랙에서 언급했던 서비차 관련된 Wrapper 에 의존성이 생기던 문제와 비슷한 현상이 벌어질 수 있을 것 같아요

예를들면 추후에 페이지 별로 ul 의 gap 이 다르게 적용된다면? 서치바를 사용할 때와 같은 문제가 발생할 것 같은데

```tsx
        <ul className={styles.searchSuggestionListSection}>
          <SearchSuggestionList relatedKeyword={relatedKeywordsData?.artists} />
        </ul>

```

이런식으로 사용하는 부모쪽에서 ul태그로 감싸는 방법은 어떠신가요??

Reply:
wrapper 관련 스타일을 clsx(cn)으로 넘겨주는건 어떤가요 ?

Reply:
우선 styleVariants 를 이용해서 base를 만들어두고 추후에 확장할 수 있게끔 하겠습니다